eclair-core/src/main/scala/fr/acinq/eclair/channel/Commitments.scala

```diff
@@ -93,7 +93,7 @@ case class Commitments(channelVersion: ChannelVersion,
 
     localCommit.spec.htlcs.collect(outgoing).filter(expired) ++
       remoteCommit.spec.htlcs.collect(incoming).filter(expired) ++
-      remoteNextCommitInfo.left.toSeq.flatMap(_.nextRemoteCommit.spec.htlcs.collect(incoming).filter(expired).toSet)
+      latestRemoteCommit.spec.htlcs.collect(incoming).filter(expired)
```

Comment:
This one shouldn't change, it becomes confusing in the case where `remoteNextCommitInfo` is a `Right`.